lib/Config.cpp

```diff
@@ -8,15 +8,38 @@ void LibConfig::Config::parseResponse(json const &data)
 {
   std::string status;
   try {
-    status = data.at("REQUEST_STATE").get<std::string>();
+    //status = data.at("REQUEST_STATE").get<std::string>();
+    status = "STILL_NO_STATUS_SENT";
```

Comment:
That's a bad name for the status, something like "NO_STATUS_SENT" would be better/

Reply:
Status is not returned yet from the service, and this message will be removed soon, so I don't think it's important, but as you wish